dungeon/src/contrib/entities/LeverFactory.java

```diff
@@ -55,10 +54,115 @@ public static Entity createLever(Point pos, ICommand onInteract) {
             }));
     return lever;
   }
+
+  /**
+   * Creates a lever entity at a given position, with a specified behavior when interacted with. The
+   * lever is initially off. The lever is interactable and can be toggled on and off.
+   *
+   * <p>The lever will use the default leaver design.
+   *
+   * @param pos The position where the lever will be created.
+   * @param onInteract The behavior when the lever is interacted with. (isOn, lever, who)
+   * @return The created lever entity.
+   * @see LeverComponent LeverComponent
+   * @see contrib.systems.LeverSystem LeverSystem
+   */
+  public static Entity createLever(Point pos, ICommand onInteract) {
+    return createLever(pos, onInteract, Design.LEAVER);
+  }
+
+  /**
+   * Creates a lever entity at a given position, with a specified behavior when interacted with. The
+   * lever is initially off. The lever is interactable and can be toggled on and off.
+   *
+   * <p>The lever will use the torch design.
+   *
+   * @param pos The position where the lever will be created.
+   * @param onInteract The behavior when the lever is interacted with. (isOn, lever, who)
+   * @return The created lever entity.
+   * @see LeverComponent LeverComponent
+   * @see contrib.systems.LeverSystem LeverSystem
+   */
+  public static Entity createTorch(Point pos, ICommand onInteract) {
+    return createLever(pos, onInteract, Design.TORCH);
+  }
+
+  /**
+   * Creates a lever entity at a given position, with a specified behavior when interacted with. The
+   * lever is initially off. The lever is interactable and can be toggled on and off.
+   *
+   * <p>The level will trigger no event on interaction, it only toggles the {@link
+   * LeverComponent#isOn()} value.
+   *
+   * <p>The lever will use the default design.
+   *
+   * @param pos The position where the lever will be created.
+   * @return The created lever entity.
+   * @see LeverComponent LeverComponent
+   * @see contrib.systems.LeverSystem LeverSystem
+   */
+  public static Entity createLever(Point pos) {
+    return createLever(pos, ICommand.EMPTY_COMMAND);
+  }
+
+  /**
+   * Creates a lever entity at a given position, with a specified behavior when interacted with. The
+   * lever is initially off. The lever is interactable and can be toggled on and off.
+   *
+   * <p>The level will trigger no event on interaction, it only toggles the {@link
+   * LeverComponent#isOn()} value.
+   *
+   * <p>The lever will use the def torch.
+   *
+   * @param pos The position where the lever will be created.
+   * @return The created lever entity.
+   * @see LeverComponent LeverComponent
+   * @see contrib.systems.LeverSystem LeverSystem
+   */
+  public static Entity createTorch(Point pos) {
+    return createTorch(pos, ICommand.EMPTY_COMMAND);
+  }
+
+  /**
+   * Represents different design types with associated texture paths. Each design has a list of
+   * textures for both "on" and "off" states.
+   */
+  public enum Design {
```

Comment:
Gehört das wirklich in die LeverFactory? Im Prinzip sollen das Kunden aufrufen, und die müssen dann immer `LeverFactory.Design` (oder so) kennen bzw. finden und schreiben? Ich hätte das Enum eher in einem Package `entities.lever` erwartet.

Reply:
Ich finde schon. 
Die Facotry nutzt es, es ist für die Factory gedacht... .
In `entities.lever` würde ich eher konkrete Schalter erwarten....

Reply:
hmmm. ich sehe deinen punkt, aber irgendwie passt das für mich trotzdem noch nicht ...

Reply:
aber egal. es ist gemerged, und wer weiß, ob und wer das enum am ende braucht ...